feat(curriculum): forum leaderboard project (#50241)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443d6ab21742c031bea7778.md

---
id: 6443d6ab21742c031bea7778
title: Step 52
challengeType: 0
dashedName: step-52
---

# --description--

> You will write another `if` statement inside the `showTime` function. This time it will check whether the constant `hoursAgo` is less than 24. If it is, then it will `return` the template literal saying `${hoursAgo}h ago`. What it does is that it will return the number of hours ago the post was created.

Reviewer: ```suggestion
Write another `if` statement inside the `showTime` function to check whether the constant `hoursAgo` is less than 24. If it is, then it will `return` the template literal `${hoursAgo}h ago`, showing the number of hours ago the post was created.
```

Author: added

# --hints--

You should create an `if` statement that checks whether the constant `hoursAgo` is less than 24.

```js
assert(code.match(/if\s*\(\s*hoursAgo\s*<\s*24\s*\)/));
```

You should use a `return` statement inside the if statement.

```js
assert(code.match(/if\s*\(\s*hoursAgo\s*<\s*24\s*\)\s*{\s*return\s*.*/));
```

> You should not use single or double quotes after the `return` statement, you should use a template literal using backticks symbol.

Reviewer: ```suggestion
You should not use single or double quotes after the `return` statement, you should use a template literal using the backticks symbol.
```

Author: added
